demos/GMW_demo/gmw_share.js

```diff
@@ -0,0 +1,182 @@
+
+/**
+ * the function to compute the shares of input (can be overridden using hooks).
+ * @ignore
+ * @function gmw_compute_share
+ * @param {module:jiff-client~JIFFClient} jiff - the jiff instance
+ * @param {number} the secret to share
+ * @param {Array} parties_list - array of party ids to share with.
+ * @param {number} threshold - the min number of parties needed to reconstruct the secret, defaults to all the receivers.
+ * @param {number} Zp - the mod.
```

Comment:
let's remove the Zp parameter (from both function definition and documentation) since it will always be Zp=2

Reply:
Maybe we can remove the threshold parameter as well? I believe in the GMW scheme everything is threshold=party_count (don't quote me on that)

Reply:
leaving threshold since another thing about gmw is the threshold with say n/2 party etc.